lib/board.rb

```diff
@@ -0,0 +1,55 @@
+
+class Board
+  attr_accessor :cells
+
+  def initialize           
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts "\n"      
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+ def full?
+    @cells.all?{|token| token == "X" || token == "O"}
+  end
+  
+ def position(input)
+    @cells[input.to_i - 1]
+  end
+
+  def turn_count
```

Comment:
@Lani1321 @klovae  for lines 31-36 we could use the ruby method `count`

example 

```ruby
@cells.count { |cell| cell == "X" || cell == "O" }
```